triage/interactive.js

```diff
@@ -130,34 +131,41 @@ function rerender(maxCount) {
 
   summaryText += ` out of ${builds.runCount} builds from ${builds.getStartTime()} to ${builds.getEndTime()}.`
 
-  summary.innerText = summaryText;
+  if (maxCount !== 0) {
+    summary.innerText = summaryText;
 
-  if (clustered.length > 0) {
-    let graph = addElement(summary, 'div');
-    renderGraph(graph, clustered.allBuilds());
-  }
+    if (clustered.length > 0) {
+      let graph = addElement(summary, 'div');
+      renderGraph(graph, clustered.allBuilds());
+    }
 
-  renderSubset(0, maxCount || 10);
+    renderSubset(0, maxCount || 10);
+  }
 
-  drawVisibleGraphs();
+  setTimeout(drawVisibleGraphs, 0);
```

Comment:
why timeout? Maybe comment the intent. Might also be better to encapulate this in a "drawQuickly" or something method which takes the drawVisibleGraphs function